code/datums/components/burning.dm

```diff
@@ -63,6 +71,37 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 
 	examine_list += span_danger("[source.p_theyre(TRUE)] burning!")
 
+/// Handles searing the hand of anyone who tries to touch this without protection.
+/datum/component/burning/proc/on_attack_hand(atom/source, mob/living/carbon/user)
+	SIGNAL_HANDLER
+
+	// attack_hand() behavior should only apply to items, for now
+	if(!isitem(source))
+		return NONE
```

Comment:
why?

Reply:
This was the old behavior. I do not believe this behavior is wanted on other atoms.